### PR TITLE
Add basic UI:School-Library:day-4

### PR DESCRIPTION
**Pull request by Shubh-Dev**
Update-2
Error fixed

Update!!
Made changes according to the Reviewer's feedback

Hello Dear Reviewer,
Added the basic UI,
Let me know If everything is alright. 

[Branch](https://github.com/Shubh-Dev/school-library/tree/add-basic-UI)

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+      puts "#{index} Name: #{item.name} Age: #{item.age} ID: #{item.id}"
+    end
+  end
+
+  def list_persons
+    list_all_people
+    puts 'Please Enter to continue'
+    gets.chomp
+    menu_option
+  end
```
Comment (DeliceLydia) [inline app.rb]: - [ ] I noticed that when I choose a number may be `list of books` you have to repeat it twice to work check here I created students and after I wanted to check `list of them` and I had to repeat 2 twice, please make sure anytime I choose one option to work without repeating it
bk

File: person.rb [hunk 2]
```diff
@@ -24,7 +24,14 @@ def can_use_services?
...
+  public :can_use_services?
+
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals.push(rental) unless @rentals.include?(rental)
+    rental.person = self
+  end
```
Comment (DeliceLydia) [inline person.rb:36]: - [ ] I am not able to create a `rental` please address that
rent

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Shubh-Dev, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Highlights for your project:
- Awesome README
- Followed git workflow


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: app.rb (hunk 1 above)
Comment (sja-thedude) [reply app.rb]: Implemented 👍

File: rental.rb [hunk 3]
```diff
@@ -1,17 +1,28 @@
...
-    book.rentals << self unless @book.rentals.include?(self)
+    @book.rentals << self unless @book.rentals.include?(self)
+  end
+
+  def person=(person)
+    @person = person
+    @person.rentals << self unless @person.rentals.include?(self)
+  end
+
+  def book=(book)
```
Comment (sja-thedude) [inline rental.rb]: - [ ] Great job in making the rental work, however, there is an error after typing the date. Kindly fix it.

Screenshot 2022-09-02 at 9 23 37 AM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Shubh-Dev, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: rental.rb [hunk 4]
```diff
@@ -1,19 +1,30 @@
...
+  # def person=(person)
+  #   @person = person
+  #   @person.rentals << self unless @person.rentals.include?(self)
+  # end
+
+  # def book=(book)
+  #   @book = book
+  #   @book.rentals << self unless @person.rentals.include?(self)
+  # end
+
```
Comment (RolandM99) [inline rental.rb]: - [ ] [OPTIONAL] Kindly, remove the commented code from your codebase. This will make your code clean.

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @Shubh-Dev, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**